test/scales/tickGeneratorsTests.ts

```diff
@@ -1,72 +1,79 @@
 ///<reference path="../testReference.ts" />
 
-describe("Tick generators", () => {
-  describe("interval", () => {
-    it("generate ticks within domain", () => {
-      let start = 0.5, end = 4.01, interval = 1;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [0.5, 1, 2, 3, 4, 4.01], "generated ticks contains all possible ticks within range");
-    });
+describe("Scales", () => {
+  describe("Tick generators", () => {
+    describe("Interval", () => {
+      let scale: Plottable.Scales.Linear;
 
-    it("domain crossing 0", () => {
-      let start = -1.5, end = 1, interval = 0.5;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [-1.5, -1, -0.5, 0, 0.5, 1], "generated all number divisible by 0.5 in domain");
-    });
+      beforeEach(() => {
+        scale = new Plottable.Scales.Linear();
+      });
 
-    it("generate ticks with reversed domain", () => {
-      let start = -2.2, end = -7.6, interval = 2.5;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [-7.6, -7.5, -5, -2.5, -2.2], "generated all ticks between lower and higher value");
-    });
+      it("generates ticks within domain", () => {
+        let start = 0.5, end = 4.01, interval = 1;
+        scale.domain([start, end]);
+        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
+        assert.deepEqual(ticks, [0.5, 1, 2, 3, 4, 4.01], "generated ticks contains all possible ticks within range");
```

Comment:
Given the name of the test, wouldn't it make more sense to assert that each value is inside the domain?

Alternatively, we could change the message.